R/mainTests.r

```diff
@@ -1232,13 +1306,13 @@ school_laestab_duplicates <- function(data) {
 
     if (length(multi_count_code) == 0 & length(multi_count_name) == 0) {
       output <- list(
-        "message" = "Every school name has one laestab, and vice versa.",
+        "message" = paste("Every", geography_matrix[13, 3], "value has one", geography_matrix[13, 4], "value, and vice versa."),
         "result" = "PASS"
       )
     } else {
       if (length(multi_count_code) == 1 & length(multi_count_name) == 0) {
         output <- list(
-          "message" = paste0("The following school_name has multiple laestabs: ", paste0(multi_count_code), ". 
+          "message" = paste0("The following ", geography_matrix[13, 3], "value has multuple", geography_matrix[13, 4], " values: ", paste0(multi_count_code), ". 
                              <br> - Every school name should have only one laestab and vice versa."),
```

Comment:
Potential issue with this check but not sure if I'm just overthinking! - e.g. we can have two legitimate "St Paul's" schools in different parts of the country with different LAESTABs - way around it is taking into account LA? But then that forces analysts to include that level info in all school files. Might be easier to just remove the part of the test that checks for name sharing a code for now? Codes sharing a name should still be highlighted though

Reply:
As we're using LAEstab we could extract the LA code from the first 3 digits of it to identify the LA. I guess we could remove the duplicate name part of these checks? There shouldn't be shared codes I don't think? Can discuss on stand-up with @lauraselby

Reply:
Resolving this as now removing these checks entirely (the platform should be fine with duplicates like these)